addons/web/tests/test_pivot_export.py

```diff
@@ -0,0 +1,34 @@
+import json
+
+from odoo import http
+from odoo.tests.common import HttpCase
+
+
+class TestPivotExport(HttpCase):
+
+    def test_export_xlsx_with_integer_column(self):
+        """ Test the export_xlsx method of the pivot controller with int columns """
+        self.authenticate('admin', 'admin')
+        jdata = {
+            'title': 'Sales Analysis',
+            'model': 'sale.report',
+            'measure_count': 1,
+            'origin_count': 1,
+            'col_group_headers': [
+                [{'title': 'Header 1', 'width': 1, 'height': 1}],
+            ],
+            'measure_headers': [],
+            'origin_headers': [],
+            'rows': [
+                {'title': 1, 'indent': 0, 'values': [{'value': 10}]},
+            ],
+        }
+        response = self.url_open(
+            '/web/pivot/export_xlsx',
+            data={
+                'data': json.dumps(jdata),
+                'csrf_token': http.Request.csrf_token(self),
+            },
+        )
+        self.assertEqual(response.status_code, 200)
```

Comment:
```suggestion
        response.raise_for_status()
```
achieve the same as asserting the status code, but will raise an exception with the response body, making it a bit easier to debug than "4xx is not equal to 200", although an exception should had been logged by the http stack already

Reply:
Thanks, didn't know about it before